src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -422,6 +421,7 @@ public IEnumerable<WorkspaceModel> Workspaces
         /// Returns authentication manager object for oxygen authentication.
         /// </summary>
         public AuthenticationManager AuthenticationManager { get; set; }
+        public object DependencyView { get; private set; }
```

Comment:
@reddyashish what is this for?

Reply:
This is not needed as well and I should have removed. Thanks for catching it.